src/pages/games/midnight-train.astro

```diff
@@ -0,0 +1,328 @@
+---
+import { Markdown } from 'astro/components';
+import BasicMeta from '../../components/BasicMeta.astro'
+import Logo from '../../components/logo/MidnightTrain.vue'
+import BackButton from '../../components/BackButton.vue'
+---
+<html lang="en">
+	<head>
+		<BasicMeta/>
+		<title>Midnight Train</title>
+		<meta name="theme-color" content="#e9e1f9">
+		<style>
+			@font-face
+			{
+				font-family: "heading";
+				src: url('https://goodbyte.games/assets/font/webfont.woff2') format('woff2'),
+					 url('https://goodbyte.games/assets/font/webfont.woff')  format('woff');
+				font-weight: bold;
+				font-style: normal;
+			}
+
+			:global(:root)
+			{
+				--grey-00: #231a37;
+				--grey-10: #30146d;
+				--grey-20: #3d2a64;
+				--grey-30: #5b26ce;
+				--grey-40: #7d50df;
+				--grey-50: #a07fe9;
+				--grey-60: #c6b1f2;
+				--grey-70: #e9e1f9;
+				--accent: #673bcb;
+			}
+
+			:global(::selection)
+			{
+				background: var(--accent);
+				color: var(--grey-70);
+			}
+
+			html
+			{
+				background: var(--grey-00);
+				color: var(--grey-70);
+				text-rendering: optimizeSpeed; 
+				-webkit-font-smoothing: antialiased;    
+				-moz-osx-font-smoothing: grayscale;
+				font-feature-settings:"kern" 1;
+				font-kerning: normal;
+				font-family: -apple-system, BlinkMacSystemFont, 'Futura', Roboto, 'Helvetica Neue', sans-serif;
+			}
+
+			body
+			{
+				margin: 0;
+				padding: 0;	
+			}
+			
+			#tagline
+			{
+				font-style: italic;
+				max-width: 400px;
+				margin: 0 auto 80px;
+				text-align: center;
+			}
+
+			main
+			{
+				padding: 0 160px;
+				box-sizing: border-box;
+			}
+
+			:global(p)
+			{
+				font-size: 14px;
+				line-height: 1.6;
+				opacity: 0.85;
+				letter-spacing: 0.03em;
+				max-width: 50rem;
+				margin: 1rem auto;
+			}
+
+			:global(h2)
+			{
+				width: 100%;
+				max-width: 50rem;
+				margin: 4rem auto 2rem;
+				font-size: 2.5rem;
+				font-family: "heading";
+				text-transform: uppercase;
+			}
+
+			:global(h3)
+			{
+				width: 100%;
+				max-width: 50rem;
+				margin: 2rem auto 0.5rem;
+				text-transform: uppercase;
+				letter-spacing: 0.05em;
+				font-size: 1.3rem;
+			}
+
+			@media screen and (max-width: 900px)
+			{
+				main
+				{
+					padding: 0 40px;
+				}
+			}
+
+			@media screen and (max-width: 600px)
+			{
+				body
+				{
+					padding-top: 40px;
+				}
+
+				main
+				{
+					padding: 0 20px;
+				}
+			}
+		</style>
+	</head>
+	<body>
+		<header>
+			<h1>
+				<Logo/>
+			</h1>
+			<p id="tagline">A framework for fluidly moving between game systems within a single, continuous narrative.</p>
+			<BackButton/>
+		</header>
```

Comment:
As fun and cool as it is to have the markdown and styles in the same file, it does make them quite large. We can move to markdown files with associated layouts, or something like that if you prefer. We can figure out what the best structure is once we figure out Astro better.